src/main/scripts/src/app/core/rest/group.service.ts

```diff
@@ -18,9 +18,14 @@
  * -----------------------------------------------------------------------/
  */
 
-export interface Entity {
+import {Injectable} from '@angular/core';
 
-  name: string;
-  roles: string[];
+import {Observable} from 'rxjs/Observable';
 
+@Injectable()
+export class GroupService {
+
+    public getGroups(): Observable<string[]> {
+        return Observable.of(['directors', 'customers']);
```

Comment:
I think this method should take `organizationCode` as a parameter since groups are defined for each organization separately.

Reply:
It should also return a list of group objects so we do not have to refactor it in the future.
```typescript
export interface Group {
  name: string;
}
```